lua/todo-comments/highlight.lua

```diff
@@ -274,8 +274,8 @@ function M.stop()
   pcall(vim.cmd, "augroup! Todo")
   M.wins = {}
   vim.fn.sign_unplace("todo-signs")
-  for _, buf in pairs(M.bufs) do
-    vim.api.nvim_buf_clear_namespace(buf, Config.ns, 0, -1)
+  for buf, _ in pairs(M.bufs) do
+    vim.api.nvim_buf_clear_namespace(buf or 0, Config.ns, 0, -1)
```

Comment:
The `or 0` is not required here either because the table keys will always be numbers.

Reply:
Fixed in 24c0877.